apps/bynder/src/index.js

```diff
@@ -57,6 +60,10 @@ const FIELD_SELECTION = `
   createdAt
   files
   originalUrl
+  textMetaproperties {
```

Comment:
Only question I have on this PR is the fields below (name, value). Do we know they always appear in the graphql schema for all customers?

Reply:
Yeah the way this is declared in the GraphQL schema from Bynder:

```graphql
type TextMetaproperty {
  name:  String!
  value:  String!
}

type Query {
  ...
  textMetaproperties: [TextMetaproperty!]!
  ...
}
```

The bang operator `TextMetaproperty!` indicates that every returned type must have a `name, value` present, the bang operator trailing the array declaration `[TextMetaproperty!]!` indicates that every entity returned must return required fields, otherwise it will return an empty array by default.